src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
> @darkbasic do you think that you will be able to add that support to angular-meteor-docs?

Probably, but first I will have to figure out why `angular-meteor-docs` is currently broken for me: not being able to properly debug it is a showstopper. I get tons of `EXCEPTION: Uncaught (in promise): Response with status: 404 null for URL` and I suspect the culprit is `generate-static.ts` from `tutorial-infrastructure`, but despite filling `generate-static.ts` with tons of catches I cannot get the exception. The stacktrace points to `zone-node.js`, but it doesn't make sense from my perspective.